khmer/kfile.py

```diff
@@ -22,16 +24,16 @@ def check_input_files(file_path, force):
     """
     mode = None
 
-    if file_path is '-':
+    if file_path == '-':
```

Comment:
wow, super subtle bug. A description of why this is wrong: http://stackoverflow.com/questions/2988017/string-comparison-in-python-is-vs
(I had to pdb and step to find it).